parsl/tests/site_tests/conda_setup.sh

```diff
@@ -0,0 +1,124 @@
+        echo "module load miniconda-3/latest;"           >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+        return
+
+    elif [[ "$(hostname)" =~ .*frontera.* ]]
+    then
+        echo "On Frontera"
+        if [[ -d ~/anaconda3 ]]
+        then
+            echo "Loading anaconda3 from ~/anaconda3"
+            source ~/anaconda3/bin/activate
+        else
+            echo "Please install conda to your home dir at ~/anaconda3"
+        fi
+        conda create -p $CONDA_TARGET python=3.7 --yes --force
+        conda activate $CONDA_TARGET
+        echo "source ~/anaconda3/bin/activate;"          >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+
+    elif [[ "$(hostname -f)" =~ .*summit.* ]]
+    then
+        echo "On Summit"
+        module load ibm-wml-ce
+        conda create -p $CONDA_TARGET --yes --force
+        conda activate $PWD/$CONDA_TARGET
+        conda install paramiko>=2.7.1 pip numpy psutil pandas --yes
+        # conda install --file requirements.txt --yes
+        echo "module load ibm-wml-ce"                    >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+
+    elif [[ "$(hostname)" =~ .*cori.* ]]
+    then
+        echo "On Cori"
+        module load python/3.7-anaconda-2019.07
+        conda create -p $CONDA_TARGET python=3.7 --yes --force
+        conda activate $PWD/$CONDA_TARGET
+        echo "module load python/3.7-anaconda-2019.07;"  >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+
+    elif [[ "$(hostname)" =~ .*stampede2.* ]]
+    then
+        echo "On Stampede2"
+        if [[ -d ~/anaconda3 ]]
+        then
+            echo "Loading anaconda3 from ~/anaconda3"
+            source ~/anaconda3/bin/activate
+        else
+            echo "Please install conda to your home dir at ~/anaconda3"
+        fi
+        conda create -p $CONDA_TARGET python=3.7 --yes --force
+        conda activate $PWD/$CONDA_TARGET
+        echo "source ~/anaconda3/bin/activate;"          >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+
+    elif [[ "$(hostname)" =~ .*comet.* ]]
+    then
+        echo "On Comet"
+        if [[ -d ~/anaconda3 ]]
+        then
+            echo "Loading anaconda3 from ~/anaconda3"
+            source ~/anaconda3/bin/activate
+        else
+            echo "Please install conda to your home dir at ~/anaconda3"
+        fi
+        conda create -p $CONDA_TARGET python=3.7 --yes --force
+        conda activate $PWD/$CONDA_TARGET
+        echo "source ~/anaconda3/bin/activate;"          >  ~/setup_parsl_test_env.sh
+        echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
+
+    else
+        echo "Treating $(hostname) as Local"
```

Comment:
This doesn't properly test the unknown system in "test batch system and running on worker nodes" sense.
So I think should give a failure rather than run a local test and give potentially false confidence. It's fine to say "we don't know how to test this system", and it's what we would say on slack when someone shows up with a question about why the site test passes even though parsl doesn't work on their site.

Reply:
Fixed.